packages/dispatchr/lib/DispatcherContext.js

```diff
@@ -14,6 +14,7 @@ var debug = require('debug')('Dispatchr:DispatcherContext');
  * @constructor
  */
 function DispatcherContext (dispatcher, context) {
+    this.context = context;
```

Comment:
We had no way of getting to `context` before, so I added it here so error handlers can use it later.